src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px; // Фиксированная ширина
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  box-sizing: border-box;
+
+  &__imac {
+    padding: 0 3px;
+    width: 160px;
+    height: 134px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), element modifiers should not set dimensions. Consider moving the width and height to the block level if they define the size of the card element or to an element if they define the size of an inner element.